loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -104,13 +105,20 @@ protected static void load(LoadFlowParameters parameters, PlatformConfig platfor
 
     private boolean t2wtSplitShuntAdmittance;
 
-    public LoadFlowParameters(VoltageInitMode voltageInitMode, boolean transformerVoltageControlOn,
-                              boolean noGeneratorReactiveLimits, boolean phaseShifterRegulationOn, boolean t2wtSplitShuntAdmittance) {
+    private boolean simulShunt;
```

Comment:
Be careful, you forgot to take into account this new attribute in many places in this class.